lib/spack/docs/packaging_guide.rst

```diff
@@ -865,6 +865,36 @@ Fetching a revision
 Subversion branches are handled as part of the directory structure, so
 you can check out a branch or tag by changing the ``url``.
 
+-----------------------------------------
+Standard repositories for python packages
+-----------------------------------------
+
+In addition to their developer websites, many python packages are hosted at the
+`Python Package Index (PyPi) <https://pypi.python.org/pypi>`_. Although links to
+these individual files are typically `generated using a hash
+<https://bitbucket.org/pypa/pypi/issues/438>`_ it is often possible to find a
+reliable link of the format
+
+.. code-block:: sh
```

Comment:
This isn't really a shell script, and shouldn't be formatted as one. I would use `none` as the format.

Aside from that, looks good! You don't have to go back in and change this, just pointing it out for future reference.